### PR TITLE
docs: `s/driver/pilote/g`

### PR DESCRIPTION
**Pull request by chadyred**

### CODE REVIEW

**Comment by devnexen** [comment]
Do you plan many more fixes in the same category ? if that s the case it might be best to regroup these as much as possible.

**Comment by chadyred** [comment]
I read the doc and I make changes on the road directly. I will clone the project and change things on my workspace next time ;)

**Comment by devnexen** [comment]
No worries take your time :)

File: reference/mysqlnd/book.xml [hunk 1]
```diff
@@ -16,8 +16,8 @@
  <preface xml:id="intro.mysqlnd">
   &reftitle.intro;
   <para>
-   Le driver natif MySQL est un remplacement pour la bibliothèque cliente
-   MySQL (libmysqlclient). Le driver natif MySQL fait parti des sources
+   Le pilote natif MySQL est un remplacement pour la bibliothèque cliente
+   MySQL (libmysqlclient). Le pilote natif MySQL fait parti des sources
    officielles PHP depuis la version 5.3.0.
```
Comment (Girgias) [inline reference/mysqlnd/book.xml:21]: Huh la doc anglaise devrait plus mentionner PHP 5

**Pull request closed without merging**